src/main/drivers/stm32/dshot_bitbang.c

```diff
@@ -504,84 +507,91 @@ static bool bbMotorConfig(IO_t io, uint8_t motorIndex, motorPwmProtocolTypes_e p
     return true;
 }
 
-static bool bbUpdateStart(void)
+static bool bbTelemetryWait(void)
+{
+    // Wait for telemetry reception to complete
+    bool telemetryPending;
+    bool telemetryWait = false;
+    const timeUs_t startTimeUs = micros();
+
+    do {
+        telemetryPending = false;
```

Comment:
```suggestion
        bool telemetryPending = false;
```

Reply:
As this is in a `do {} while();` loop the above would put `telemetryPending` out of scope.

Reply:
Sorry, my fault.